app/views/events/index.html.erb

```diff
@@ -1,2 +1,57 @@
-<h1>Events#index</h1>
-<p>Find me in app/views/events/index.html.erb</p>
+<% if user_signed_in? %>
+    <%= link_to  "Log out",  destroy_user_session_path, method: :delete %>
+<% else %>
+    <%= link_to  "Log in",  new_user_session_path%>
+    <%= link_to  "Sign up",  new_user_registration_path%>
+<% end %>
+
+<h1>All events</h1>
+
+<div>
+<%= link_to "Create an event", new_event_path%>
+<h3>Upcoming events</h3>
+    <table>
+        <tr>
+            <th>Title</th>
+            <th>Date</th>
+            <th>Location</th>
+            <th colspan='1'></th>
+        </tr>
+            <%Event.upcoming.each do |event|%>
+        <tr>
+            <td><%= event.title%></td>
+            <td><%= event.date%></td>
+            <td><%= event.location%></td>
+            <td><%= link_to "See more", event_path(event)%>
+        </tr>
+            <%end%>
+    </table>  
+<h3>Past events</h3>
+            <table>
+                    
+
+                <tr>
+                    <th>Title</th>
+                    <th>Date</th>
+                    <th>Location</th>
+                </tr>
+                    <%Event.past.each do |event|%>
```

Comment:
- [ ] List past events, i.e. `@past_events = Event.upcoming_events` in your controller and use `@ past_events ` in your view ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/past_upcoming_events_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))